dists/macosx/Info.plist

```diff
@@ -4,6 +4,18 @@
 <dict>
 	<key>CFBundleDevelopmentRegion</key>
 	<string>English</string>
+	<key>CFBundleDisplayName</key>
+	<string>ScummVM</string>
+	<key>CFBundleExecutable</key>
+	<string>scummvm</string>
+	<key>CFBundleGetInfoString</key>
+	<string>1.8.0git, Copyright 2001-2016 The ScummVM Team</string>
+	<key>CFBundleIconFile</key>
+	<string>scummvm.icns</string>
+	<key>CFBundleIdentifier</key>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
```

Comment:
Yes, now that I know about the `update-version.pl` tool, I'll take care of these mistakes.

By the way, the latest Xcode requires the Info.plist to not hardcore the bundle identifier, but rather use the macro which will be replaced by a macro preprocessor during the packaging of the application. So, I decided to do the same for OS X as iOS, i.e., let the `$(PRODUCT_BUNDLE_IDENTIFIER)` macro in the template, and add a `sed` command in the `bundle` target of the makefile.